src/js/components/DateInput/__tests__/DateInput-test.tsx

```diff
@@ -1164,4 +1164,41 @@ describe('DateInput', () => {
 
     expect(input).toHaveValue('');
   });
+
+  test('read only', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <DateInput
+          format="mm/dd/yyyy"
+          value="01/01/2024"
+          readOnly
+          aria-readonly
```

Comment:
did we talk about automatically putting `aria-readonly` if `readOnly` is passed seems like a user would always want this here or we can leave it up to them

Reply:
I was considering adding it automatically but I think it only needs to be added if something isn't a semantic html form element. I forgot to remove it from my story though, I'll go back and do that.

More details:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-readonly
> Note: When using semantic HTML form controls, if you set the readonly attribute, you don't need to include aria-readonly="true".

Reply:
ok thanks 👍